packages/shared/src/contexts/AuthContext.tsx

```diff
@@ -132,6 +134,7 @@ export const AuthContextProvider = ({
 
   const authContext: AuthContextData = useMemo(
     () => ({
+      isAuthReady: !isNullOrUndefined(firstLoad),
```

Comment:
We already have `isFetched` or `loadedUserFromCache` for that right (depends on the case)?  What's the difference?

Reply:
There is a state inside the boot provider that passes the value of that query which is tied to this `firstLoad` property, so to get the most accurate it would probably be best to use this parameter.